metadata-load/src/test/resources/study/studyDocumentAPI

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please name the new test files appropriately, with the `.xml` extension and either camel case (StudyDocumentApi.xml) or underscores (study_document_api.xml).